docs/v1.0/in_forward.txt

```diff
@@ -246,6 +246,55 @@ Multiple messages may be sent in the same connection.
 
 For more details, see [Fluentd Forward Protocol Specification (v1)](https://github.com/fluent/fluentd/wiki/Forward-Protocol-Specification-v1).
 
+## Tips & Tricks
+
+### How to enable TLS/SSL encryption
+
+Since v0.14.12, Fluentd includes a built-in TLS/SSL support. Here we present a quick up and running tutorial for setting up TLS/SSL encryption:
+
+First, generate a self-signed certificate using the following command:
+
+    :::term
+    $ openssl req -new -x509 -sha256 -days 1095 -newkey rsa:2048 \
+                  -keyout fluentd.key -out fluentd.crt
+    # Note that during the generation, you will be asked for:
+    #  - a password (to encrypt the private key), and
+    #  - subject information (to be included in the certificate)
+
```

Comment:
We can use fluent-ca-generate since Fluentd v1.1.0.

Reply:
See the thread by @cosmos0920: https://github.com/fluent/fluentd-docs/pull/439#pullrequestreview-93193418